packages/components/src/color-edit/index.js

```diff
@@ -49,12 +49,14 @@ function ColorNameInput( { value, onChange } ) {
 }
 
 function ColorOption( {
+	canOnlyChangeValues,
 	color,
 	onChange,
 	isEditing,
 	onStartEditing,
 	onRemove,
 	onStopEditing,
+	slugPrefix,
```

Comment:
I'm not very keen on making the ColorEdit component aware of the prefix for custom colors. For components, do we usually expose "hooks" for consumers to tweak the normal behavior? Like, for example, we could have an `onNameChange` callback that consumers can use to tweak the name as they like (add a prefix, etc).

Not really familiar with how/whether this is a practice we want our components to have, so I'll go with whatever you think is best.

Reply:
I think the prefix may end up being used for other things besides custom colors, and it seems we will probably not need other name manipulation than the prefix. So I guess slugPrefix is enough. In the future, I'm not totally sure of the API and the place where this component will be. I guess when we remove experimental from this component or move the component to another package we should audit its API.